onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.cc

```diff
@@ -248,6 +248,12 @@ std::unique_lock<OrtMutex> TensorrtExecutionProvider::GetApiLock() const {
   return std::unique_lock<OrtMutex>(singleton);
 }
 
+#ifdef ORT_RUN_EXTERNAL_ONNX_TESTS
```

Comment:
Just wondering where this MACRO will be defined. Is it defined in build script or cmake file?

Reply:
in cmake.  when you set run_onnx_tests option in build script the cmake option gets enabled and that macro gets defined.

Reply:
see https://github.com/microsoft/onnxruntime/blob/963e1ace4ed21fd7e368a97b077fc474ff551022/cmake/CMakeLists.txt#L1534

Reply:
Okay, I see.